guides/tutorials/codelabs/halyard-getting-started/index.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: single
+title:  "Getting Started with Halyard"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In this codelab, you will learn the basics of [halyard](/setup/install/halyard/), Spinnaker's tool for managing your Spinnaker instance. 
+
+## Overview
+
+In our scenario, we want to create a Spinnaker instance and set it up as follows:
+
+* The Spinnaker instance is itself running in a Kubernetes cluster
+* The Kubernetes provider is set up so that we can deploy our custom apps to Kubernetes
+* We can pull Docker images from our Google Container Registry
+* We use GCS as our persistence store
+
+For this exercise we will be operating entirely within one GCP project, and use Google Container Engine (GKE) as our Kubernetes cluster. 
+
+
+## Part 0: Prerequisites
+
+We assume the reader has some prior experience with Kubernetes and GCP, but will also provide some guidance along the way. First, you'll need to [install gcloud](https://cloud.google.com/sdk/downloads#interactive) and have a [GKE](https://console.cloud.google.com/kubernetes/list)) cluster.
+
+
+## Part 1: Install halyard
+
+### Create halyard host VM
+
+```
+HALHOST=$USER-halyard-`date +%m%d`
+
+gcloud compute instances create $HALHOST \
+    --scopes storage-full,compute-rw \
+    --project={YOUR_GCP_PROJECT} \
```

Comment:
I think it's OK to drop this, or to add a call like this

```
PROJECT=$(gcloud info --format='value(config.project)')
```

to make this guide more copy/paste-able